Bug/154213269/fix langing page

bahdcoder opened the pull request:
### What does this PR do?

- Fixes the AOS plugin bug

### Description of Task to be completed?

- The team avatars and how-to-play description were not loading after merge, this pull request fixes that by configuring the AOS plugin correctly

### How should this be manually tested?

- Run npm start to start the server
- Go to localhost:3000 to see the landing page



### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- # 154213269

### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-01-08 at 4 33 51 pm" src="https://user-images.githubusercontent.com/25517388/34715498-2ebd416c-f52d-11e7-9fe5-8951c3aaea72.png">

### 
Questions:

N/A

Commit:
bug(aos-plugin): fix on scroll effect on home page

fix the text on the `how to play` section
configure the AOS plugin correctly

[Finishes #154213269]

Commit:
bug(integration tests): fix tests for signUp and signIn

Commit:
bug(integration tests): fix tests for signUp and signIn

Commit:
bug(integration tests): fix tests for signUp and signIn

Commit:
bug(integration tests): fix tests for signUp and signIn

Commit:
bug(integration tests): fix tests for signUp and signIn

Commit:
bug(integration tests): fix tests for signUp and signIn

chinezenwosu commented on app/controllers/index.js:
```diff
@@ -1,23 +1,23 @@
+ * @returns {redirect} redirect user
  */
-exports.play = function(req, res) {
+exports.play = (req, res) => {
```
Since you have babel now, you can change this to es6 `export`. Change all instances in this PR.

chinezenwosu commented on app/controllers/users.js:
```diff
@@ -1,9 +1,12 @@
-var mongoose = require('mongoose'),
+const { isEmail } = require('validator');
+const mongoose = require('mongoose'),
+  jwt = require('jsonwebtoken'),
```
Split this into multiple declarations.

chinezenwosu commented on app/controllers/users.js:
```diff
@@ -74,37 +77,28 @@ exports.checkAvatar = function(req, res) {
+exports.create = (req, res) => {
+  const user = new User(req.body);
+  user.provider = 'local';
+
```
Remove space.

chinezenwosu commented on app/views/includes/head.jade:
```diff
@@ -1,33 +1,34 @@
-  meta(charset='utf-8')
-  meta(http-equiv='X-UA-Compatible', content='IE=edge,chrome=1')
-  meta(name='viewport', content='width=device-width,initial-scale=1')
+//- head
```
Is this still going to be used? If not, remove completely.

Commit:
chore(babel): add babel compilation

Commit:
chore(babel): add babel compilation

Commit:
fix import extensions error

chinezenwosu commented on app/controllers/avatars.js:
```diff
@@ -1,29 +1,8 @@
-           '/img/chosen/N03.png',
-           '/img/chosen/N04.png',
-           '/img/chosen/N05.png'];
+const avatars = ['/img/chosen/E01.png', '/img/chosen/F01.png', '/img/chosen/FA04.png', '/img/chosen/FB03.png', '/img/chosen/FC01.png', '/img/chosen/FD01.png', '/img/chosen/FE01.png', '/img/chosen/FH03.png', '/img/chosen/FI02.png', '/img/chosen/H01.png', '/img/chosen/J01.png', '/img/chosen/M05.png', '/img/chosen/N02.png', '/img/chosen/N03.png', '/img/chosen/N04.png', '/img/chosen/N05.png'];
```
Please leave on separate lines as they were.

Commit:
use es6 import and export statements

chinezenwosu commented on gulpfile.babel.js:
```diff
@@ -1,18 +1,34 @@
 });
 
+gulp.task('watch', () => {
+  gulp.watch('app/views/**', browserSync.reload());
```
Watch for all files in `public` and `app` using `public/**/....`

Commit:
chore(configure-mocha-babel): setup mocha and babel

add babel compiler to mocha tests
fix failing tests due to babel compiler

Commit:
Merge branch 'bug/153741758/fix-langing-page' of https://github.com/andela/blade-runner-cfh into bug/153741758/fix-langing-page

Add changes from upstream

Commit:
chore(configure-mocha-babel): setup mocha and babel

add babel compiler to mocha tests
fix failing tests due to babel compiler
[Finishes #153741758]

Commit:
chore(configure-mocha-babel): setup mocha and babel

add babel compiler to mocha tests
fix failing tests due to babel compiler
